db/dcrsqlite/sync.go

```diff
@@ -405,3 +409,54 @@ func (db *wiredDB) getBlock(ind int64) (*dcrutil.Block, *chainhash.Hash, error)
 
 	return block, blockhash, nil
 }
+
+// Import all side chains. Similar to pgblockchain.MissingSideChainBlocks
```

Comment:
Please begin all function docs, especially for exported functions, with the function name (i.e. `// ImportSideChains ...`).  This is just convention, not my silly rule.  :)
https://blog.golang.org/godoc-documenting-go-code
https://golang.org/doc/effective_go.html#commentary